web/src/OverviewResourcePane.tsx

```diff
@@ -25,7 +25,8 @@ let OverviewResourcePaneRoot = styled.div`
 let Main = styled.div`
   display: flex;
   width: 100%;
-  flex: 1 1 auto;
+  // In Safari, flex-basis "auto" squishes OverviewTabBar + OverviewResourceBar
+  flex: 1 1 100%;
```

Comment:
I did this to fix an issue in my version of Safari (kinda outdated...13.1.2). I'm not 100% sure what the bug is, but this appears to fix the layout issue without causing adverse effects on my other browsers.